### ISSUE TITLE
Add support for alt text for media when publishing 

### ISSUE THREAD

**Issue by rabble** [planetary-social/nos#1226]
As a user of nos I should be able to add alt text for any media I publish and to make alt text available for accessibility purposes. 

![IMG_1230](https://github.com/planetary-social/nos/assets/76/f2d12342-f418-4bc5-920a-0516de1fcec2)
